recipe_scrapers/kitchensanctuary.py

```diff
@@ -0,0 +1,78 @@
+# mypy: allow-untyped-defs
+
+from recipe_scrapers._grouping_utils import IngredientGroup
```

Comment:
```suggestion
from recipe_scrapers._grouping_utils import group_ingredients
```

Related to other comment